### ISSUE TITLE
Pull request limitation

### ISSUE THREAD

**Issue by ColinAE** [geanders/futureheatwaves#37]
Dr. Anderson, 

Github only seems to be letting me issue a single pull request at a time. I have pending updates for the C++ part. You probably noticed that I did not merge your updates with mine before submitting the request. Let me know if you want me to merge your updates with my version and resubmit the pull request.

**Comment by geanders** [comment]
Should be good now. Sorry about the delay-- I've been in Beijing this week. I'll be back next week. I think the only big difference between the two branches was there was a place in `main.R` where you still were checking with `grepl(".csv", ....)` for an error in format and I think it should be `!grepl(".csv",...)`. You might want to double check and make sure it's working like we want at that part now.

**Comment by ColinAE** [comment]
Let me finish up what I've been working on and then I will submit another pull request sometime tomorrow or the next day. After that, I will wait on you to merge so I can sync our projects. I think we can finish the code by the end of this week.

**Comment by ColinAE** [comment]
I noticed that you already had my C++ updates in the repository. I'm curious as to how they got there, since I don't believe I had uploaded them upon submitting that first pull request. Did the system put my updates into the pull request automatically?

**Comment by geanders** [comment]
Hmmm... I'm not sure. Perhaps so-- maybe it pulls your repo as it exists when I choose to pull rather than when you submitted the request. Although that seems more error prone than the alternative, doesn't it.
